### PR TITLE
Add quotes around linter command args

### PR DESCRIPTION
**Pull request by millerds**
This will fix the problem with the lint command when projects have a space in the name or path and for the linter running on Mac.

### CODE REVIEW

File: packages/office-addin-lint/src/lint.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ const eslintConfigPath = path.resolve(__dirname, "../config/.eslintrc.json");
 const eslintTestConfigPath = path.resolve(__dirname, "../config/.eslintrc.test.json");
 
 function execCommand(command: string) {
+  console.log(`> ${command}`);
```
Comment (igor-ribeiiro) [inline packages/office-addin-lint/src/lint.ts:18]: What is this about?

Comment (millerds) [reply packages/office-addin-lint/src/lint.ts:18]: This just adds the actual command that was run as part of the output.  It's useful to see the paths that are part of the command . . . but maybe not necessary.

Comment (akrantz) [reply packages/office-addin-lint/src/lint.ts:18]: I think you should take this out.

**Comment by millerds** [comment]
The mac test pass locally on my macbook pro.  I think it's a problem with the CI/CD machine (I'm currently trying to set up a new one).

**Review by igor-ribeiiro — APPROVED** [review]

**Pull request merged**